django/santropolFeast/member/management/commands/processscheduledstatuschange.py

```diff
@@ -0,0 +1,46 @@
+from django.core.management.base import BaseCommand
+from member.factories import ClientFactory
+from member.models import Client, ClientScheduledStatus, Member
+from django.core.management import call_command
+import os
+import datetime
+from sys import path
+
+
+class Command(BaseCommand):
+    help = 'Process scheduled status changes, \
+        queued in «member_ClientScheduledStatus» table.'
+
+    def handle(self, *args, **options):
+        # List all change not processed, and older or equal to now
+        changes = ClientScheduledStatus.objects.filter(
+            operation_status=ClientScheduledStatus.TOBEPROCESSED
+        ).filter(
+            change_date__lte=datetime.date.today()
+        )
+
+        # For each change to be processed,
+        for statusChange in changes:
```

Comment:
You should use underscores, not camelCase, for variables, functions and method names. [Reference](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/)